### ISSUE TITLE
sofa-boot中支持哪几种全局配置？

### ISSUE THREAD

**Issue by wangxiaotao00** [alipay/sofa-rpc-boot-projects#37]
是否支持注解方式获取配置文件application.properties中配置项？
是否支持全局日志级别的配置，如在application.properties增加日志级别为info

**Comment by leizhiyuan** [comment]
注解方式spring boot 原生支持

```
@Value("${spelDefault.value}")

```
类似如上的注解打在属性上即可,可以测试看下.

**Comment by NeGnail** [comment]
文档里面有讲到支持的全局配置  https://github.com/alipay/sofa-rpc-boot-projects/wiki/Settings

**Comment by leizhiyuan** [comment]
看了一下代码, starter 的能设置.但是 rpc 自身的是不能设置修改的. 需要starter 从配置文件中获取并 set 到 jvm 的 property 中,然后
 rpc 日志初始化的时候,从System.getProperty 取到这个信息,才能完成从application.properties中读取并设置的过程 @guanchao-yang 帮忙看下是不是这样?

**Comment by leizhiyuan** [comment]
测试了一下,之前的版本应该都是只允许用户通过-D 参数设置的.我提个 pr. 支持一下转换这个.

**Comment by guanchao-yang** [comment]
@khotyn @leizhiyuan  是的。starter 读取的配置文件转换到系统属性里面，然后 RPC 应该识别的是这个系统属性，不是一 starter 的话，就需要配置在系统属性里面，这里面单独使用的时候需要转换一下。

**Comment by khotyn** [comment]
@guanchao-yang @leizhiyuan 建议往标准的 Spring 的配置方式靠齐。

**Comment by leizhiyuan** [comment]
@guanchao-yang @wangxiaotao00  那使用方的自己的应用日志级别这个,其实我们是没法控制的对吧.他们需要自己来读取对应的 key之类的.

**Comment by khotyn** [comment]
应用自己的不在 sofa-common-tools 的范围内。Spring Boot 本身应该提供了手段了。

**Comment by leizhiyuan** [comment]
@wangxiaotao00 @khotyn 

 看了一下 spring boot 的说明

对于日志的配置,都是类似这样的.其中的 com.xxx 换成自己的包名即可

@wangxiaotao00 麻烦试下.这个应该跟框架是没有耦合的
```
logging.level.com.xxx=DEBUG
```

**Comment by leizhiyuan** [comment]
@wangxiaotao00 如果问题解决了.可以关下 issue
